app/controllers/ncTopics.controllers.js

```diff
@@ -1,10 +1,21 @@
-const { fetchTopics} = require("../models/ncTopics.models")
+const { fetchTopics, fetchApi} = require("../models/ncTopics.models")
+
 
 exports.getTopics = (req, res, next) => {
   fetchTopics()
   .then((topics) => {
     res.status(200).send({ topics });
-    console.log(topics , '<<<<<< in controller :D ');
+  })
+  .catch((err) => {
+    console.log(err.status);
```

Comment:
if you're passing errors to your error handling middleware, I would argue that you don't need to log any errors here (your middleware can potentially do that for you if needed)